File: ios/SVGRenderNativeFormat/SVGImageViewManager.m
//
//  SVGImageViewManager.m
//  SVGRenderNativeFormat
//
//  Created by Susan Thapa on 13/07/2023.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(SvgImageView, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(svgComponent, NSDictionary)
@end

File: ios/SVGRenderNativeFormat/SVGImageViewManager.swift
//
//  SVGImageViewManager.swift
//  SVGRenderNativeFormat
//
//  Created by Susan Thapa on 13/07/2023.
//

import Foundation

@objc(SvgImageView)
class SVGImageViewManager: RCTViewManager {
  
  override class func requiresMainQueueSetup() -> Bool {
    true
  }
  
  override func view() -> UIView! {
    return SVGImageView(frame: .zero, bridge: bridge)
  }
}
